examples/iframe.html

```diff
@@ -0,0 +1,6 @@
+<iframe
+    src="./advanced.html"
+    width="736"
+    height="531"
+    allow="fullscreen 'none'"
```

Comment:
this is the syntax for disallowing a particular feature.

Reply:
I think I need to explicitly disallow it because otherwise the iframe is from the same domain and allows it by default?

Reply:
Makes sense. Interesting how the syntax of this API is different from something like controlslist